lowell-dashboard/app/templates/navbar.html

```diff
@@ -26,12 +26,17 @@
                 <li>
                     <a href="/news">News <i class="far fa-newspaper nav-icon"></i></a>
                 </li>
+                {% if current_user.is_authenticated %} <!-- Will later change to if user is admin -->
+                <li>
```

Comment:
we can allow users to create posts but must be approved by admins. I can fix this thing up.